### PR TITLE
New example program that shows the Hybrid OMPL/TrajOpt planner class

### PR DESCRIPTION
**Pull request by pbeeson**
Basically just the freespace OMPL example but using the OMPL followed by TrajOpt class

### CODE REVIEW

File: tesseract_ros/tesseract_examples/tesseract_ros_examples/include/tesseract_ros_examples/freespace_hybrid_example.h [hunk 1]
```diff
@@ -0,0 +1,63 @@
+/**
+ * @file freespace_hybrid_example.h
+ * @brief An example of a feespace motion planning with OMPL then TrajOpt
+ *
+ * @author Levi Armstrong
```
Comment (Levi-Armstrong) [inline tesseract_ros/tesseract_examples/tesseract_ros_examples/include/tesseract_ros_examples/freespace_hybrid_example.h:5]: Do you want add your name as the author?

File: tesseract_ros/tesseract_examples/tesseract_ros_examples/src/freespace_hybrid_example.cpp [hunk 2]
```diff
@@ -0,0 +1,206 @@
+/**
+ * @file freespace_hybrid_example.cpp
+ * @brief An example of a feespace motion planning with OMPL then TrajOpt.
+ *
+ * @author Levi Armstrong
```
Comment (Levi-Armstrong) [inline tesseract_ros/tesseract_examples/tesseract_ros_examples/src/freespace_hybrid_example.cpp:5]: Same here.

File: tesseract_ros/tesseract_examples/tesseract_ros_examples/src/freespace_hybrid_example_node.cpp [hunk 3]
```diff
@@ -0,0 +1,50 @@
+/**
+ * @file freespace_hybrid_example_node.cpp
+ * @brief An example of a feespace motion planning with OMPL then TrajOpt.
+ *
+ * @author Levi Armstrong
```
Comment (Levi-Armstrong) [inline tesseract_ros/tesseract_examples/tesseract_ros_examples/src/freespace_hybrid_example_node.cpp:5]: same here

**Comment by pbeeson** [comment]
I don't really care.  It's largely your example that I modified.  I don't
need the credit

**Pull request merged**